master/coverage.py

```diff
@@ -123,15 +135,19 @@
         command=[Interpolate("julia-%(prop:shortcommit)s/bin/julia"), "-e", analyze_cov_cmd]
     ),
     ShellCommand(
-        name="Move coverage.jld -> coverage_noninlined.jld",
-        command=["mv", Interpolate("julia-%(prop:shortcommit)s/share/julia/coverage.jld"), Interpolate("julia-%(prop:shortcommit)s/share/julia/coverage_noninlined.jld")]
+        name="Move coverage_coveralls.jld -> coverage_coveralls_noninlined.jld",
+        command=["mv", Interpolate("julia-%(prop:shortcommit)s/share/julia/coverage_coveralls.jld"), Interpolate("julia-%(prop:shortcommit)s/share/julia/coverage_coveralls_noninlined.jld")]
```

Comment:
Is this supposed to just be `coveralls.jld`? Similarly with `codecov.jld` below?